src/appConstants.js

```diff
@@ -55,7 +55,11 @@ const optionalEnvVars = [
   'PORT',
   'RECRUITMENT_BANNER_TEXT',
   'RECRUITMENT_BANNER_LINK',
-  'SENTRY_DSN_LEGACY'
+  'SENTRY_DSN_LEGACY',
+  'FLOATING_BANNER_DELAY',
+  'FLOATING_BANNER_LINK',
+  'FLOATING_BANNER_TYPE',
+  'FLOATING_BANNER_PAGES'
```

Comment:
barely even a nit: If you move these new FLOATING_BANNER_* keys up to L47.5 it would retain a rough A-Z ordering and wouldn't need that comma diff on L58.

Reply:
I like alphabetic order but let’s not make it a _rough_ order: [b80af60](https://github.com/mozilla/blurts-server/pull/3067/commits/b80af6076884c9a0375c3942c9395daa89a25091). ;)